AirlineBookingApp/UserManagement/Manager.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AirlineBookingApp.UserManagement
+{
+    internal abstract class Manager : User
+    {
```

Comment:
let's separate the models classes `entities` from the behavior classes `abstractions or interfaces`